_events/workshops/event-017.md

```diff
@@ -32,6 +32,9 @@ time:
 prerequisites: "no specific knowledge/skills required"
 date: 2020-08-18
 last_modified_at: 2020-09-08
+registration_url:
+    - https://indico.scc.kit.edu/event/923/
+    - https://indico.scc.kit.edu/event/922/
```

Comment:
```suggestion
    - https://indico.scc.kit.edu/event/924/
    - https://indico.scc.kit.edu/event/925/
```

Reply:
I just created the events, so we can pick the correct registration urls already.